recipes/beyla-service-graph/collector-config.yaml

```diff
@@ -53,47 +48,63 @@ spec:
         detectors: [env, gcp]
         timeout: 2s
         override: false
-      # Move server.address from a metric attribute to a resource attribute so we can use it to get k8s attributes.
+      # Move server and client address from metric attributes to resource attributes so we can
+      # use it to get k8s.pod.name for the server and client pods
       groupbyattrs:
         keys:
           - server.address
-      # Assume server.address is a pod IP address
-      resource:
-        attributes:
-          - key: k8s.pod.ip
-            from_attribute: server.address
-            action: insert
-      k8sattributes:
+          - client.address
+      transform/serveraddress_to_k8spodip:
+        metric_statements:
+          - context: resource
+            statements:
+            - delete_key(attributes, "k8s.pod.name")
+            - set(attributes["k8s.pod.ip"], attributes["server.address"])
+      # Add k8s.pod.name
+      k8sattributes/1:
```

Comment:
this looks the same as k8sattributes/2.  Can you not use the same processor twice in a single pipeline?

Reply:
Unfortunately no it fails at runtime [see here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/design.md#:~:text=The%20same%20name%20of%20the%20processor%20MUST%20NOT%20be%20referenced%20multiple%20times%20in%20the%20processors%20key%20of%20a%20single%20pipeline). I can add a comment explaining it

Reply:
A comment would be helpful

Reply:
Updated PTAL